dotcom-rendering/src/components/CardCommentCount.importable.tsx

```diff
@@ -83,12 +87,18 @@ export const CardCommentCount = ({
 				<div css={svgStyles(isDynamo, isOnwardContent)}>
 					<CommentIcon />
 				</div>
-				<div css={longStyles} aria-hidden="true">
-					{long}
-				</div>
+				<div css={longStyles}>{long}</div>
 				<div css={shortStyles} aria-hidden="true">
 					{short}
 				</div>
+				<span
+					css={css`
+						${visuallyHidden}
+					`}
```

Comment:
Does this work?

```suggestion
					css={visuallyHidden}
```

Reply:
sadly not; I tried it originally but I got this error in the browser from Emotion:

```
Error: Strings are not allowed as css prop values, please wrap it in a css template literal from '@emotion/react' like this: css`
	position: absolute;
	overflow: hidden; /* gets rid of horizontal scrollbar that appears in some circumstances */
	white-space: nowrap; /* The white-space property forces the content to render on one line. */
	width: 1px;  /* ensures content is announced by VoiceOver. */
	height: 1px; /* ensures content is announced by VoiceOver. */
	margin: -1px; /* hide or clip content that does not fit into a 1-pixel visible area. */
	padding: 0; /* hide or clip content that does not fit into a 1-pixel visible area. */
	border: 0;
	clip: rect(1px, 1px, 1px, 1px); /* clip removes any visible trace of the element */
	-webkit-clip-path: inset(50%); /* clip removes any visible trace of the element */
	clip-path: inset(50%); /* clip removes any visible trace of the element */
```

Reply:
if there's a preferred alternative for visually hiding content on DCR though I'm happy to refactor :)

Reply:
Generally we favour static styles, but there's no written recommendation or linting rule for this so up to you I think!